app/view/LayerTree.js

```diff
@@ -28,6 +28,16 @@ Ext.define('CpsiMapview.view.LayerTree', {
     hideHeaders: true,
     lines: false,
     flex: 1,
+    header: {
+        dock: 'bottom'
+    },
+    tools: [{
```

Comment:
Something like 

```
bbar: [{
  text: 'Add External Map Layer',
...
}]
```
should do the same and would make 

```
 header: {
        dock: 'bottom'
    },
```
obsolete.

Just for information, no need to change.